### PR TITLE
Add slider for time in animation controller

### PR DESCRIPTION
**Pull request by babsey**
This PR adds time slider for animation controller.

The slider for current time:
screenshot-localhost_8001-2021 12 07-14_57_11

### CODE REVIEW

**Comment by babsey** [comment]
screenshot-localhost_8001-2021 12 07-14_57_11

**Review by jens-br — CHANGES_REQUESTED** [review]
Small adjustments suggested. I added a PR with also a visual improvement in your fork.

File: src/components/parameter/ParameterEdit.vue [hunk 1]
```diff
@@ -269,7 +270,16 @@
                     <v-divider />
                     <v-row class="mx-0 py-1">
                       <v-col class="text-center" cols="2">
-                        <v-icon large right v-text="'mdi-alert-outline'" />
+                        <v-icon
+                          :small="state.options.iconSize != null ? true : false"
+                          :large="state.options.iconSize != null ? false : true"
```
Comment (jens-br) [inline src/components/parameter/ParameterEdit.vue:275]: My bad -as suggested, this should be a bit better:

```suggestion
                          :small="state.options.iconSize === 'small'"
                          :large="state.options.iconSize === 'large'"
```

**Review by jens-br — APPROVED** [review]
LGTM :+1:

**Pull request merged**